website/data/tools/babel_node_debug/usage.md

```diff
@@ -8,9 +8,9 @@ Or use the alias:
 bode-debug path/to/script.js
 ```
 
-<blockquote class="alert alert--info">
+:::info
```

Comment:
Can you check if this is working? The `/data/**/*.md` are rendered by react-markdown, which may not enable the remark plugin required by admonitions `:::`.

Reply:
<img width="1322" alt="Screenshot 2023-02-25 at 12 54 02 AM" src="https://user-images.githubusercontent.com/32865581/221272558-43bd459b-42d5-44ed-8f55-27b2f8134fa4.png">
Interestingly it seems to work 😅

Reply:
Can you check https://deploy-preview-2731--babel.netlify.app/setup#installation and click jspm?

These files do not support hot loading because they are not declared as external webpack dependencies.

Reply:
Then is it fine to replace only inside the docs and blog directory?